bin/juxtaposer/README.md

```diff
@@ -44,6 +44,180 @@ Note: More comparison types may be added in the future.
 
 ---
 
+## Running the code
+
+You can run and test this code on your machine easily with a few things:
+- [Docker](https://docker.com)
+- [Golang](https://golang.org/dl/) 1.12 or higher
+- [Secretless-broker](https://github.com/cyberark/secretless-broker) (optional)
+- A writable folder for shared socket files (e.g. `/sock` in these configurations)
+
+### Start the test databases
+
+#### PG
+
+```
+docker run --name pg-test-db \
+           -p 5433:5432 \
+           -e POSTGRES_USER=myuser \
+           -e POSTGRES_PASSWORD=mypassword \
+           -e POSTGRES_DB=mydb \
+           -d \
+           postgres
+```
+
+#### MySQL
+
+```
+docker run --name mysql-test-db \
+           -p 3307:3306 \
+           -e MYSQL_ROOT_PASSWORD=myrootpassword \
+           -e MYSQL_USER=myuser \
+           -e MYSQL_PASSWORD=mypassword \
+           -e MYSQL_DATABASE=mydb \
+           -d \
+           mysql:5
+```
+
+### Run Secretless Broker
+
+_Note: This step is optional but it is required for this specific example since it tests the broker's
+performance._
+
+- Clone the [secretless-broker repository](https://github.com/cyberark/secretless-broker) (`git clone https://github.com/cyberark/secretless-broker`).
+- Create the shared socket folder if it's missing (`sudo mkdir /sock`).
+- Create the following `secretless.yml` file in that folder:
+<details>
+  <summary><code>secretless.yml</code></summary>
+  <p>
+
+```yaml
+listeners:
```

Comment:
how come we're using a v1 config here?

Reply:
This is my local config - I'll change to v2 in followup commit